Fixed ws-url and needed steps.

linkztream opened the pull request:
Modified the ws-url to reflect the documentation from SPC-GATEWAY manual and added the two needed changes to customize.yaml + configuration.yaml to make the sensors produce usable output.

**Description:**


**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

linkztream committed:
Fixed ws-url and needed steps.

Modified the ws-url to reflect the documentation from SPC-GATEWAY manual and added the two needed changes to customize.yaml + configuration.yaml to make the sensors produce usable output.

frenck commented on source/_integrations/spc.markdown:
```diff
@@ -34,12 +34,25 @@ api_url:
 {% endconfiguration %}
 
-Supported sensors will be automatically discovered and added, however they will be hidden by default.
+Supported sensors will be automatically discovered and added, however they will be hidden by default, and unless unhidden they will not produce any history or work in automations.
```
Let's remove this part altogether.
With Lovelace, there is no reason to hide these sensors.

home-assistant/home-assistant#30699

linkztream replied:
The reason for the change was that the integration doesn't work unless the sensors are unhidden. The integration sets them to hidden, and just changing them in Lovelace reverts back to hidden at restart. At the very least we need to keep the information about the sensors not working unless they are manually unhidden. 

This was the only way I could make it work consistently.

frenck replied:
@linkztream If you've clicked the link I've put in my last message, you would have found I have changed the integration in Home Assistant to not hide those sensors anymore.

Please remove the customization additions, as they are not needed anymore.

linkztream replied:
You're absolutely right, my bad. Sorry for not reading properly! This was my first pull request ever, so got a little bit lost in it all.

With that change to the integration (and some better reading comprehension on my part) I completely agree with you.

frenck reviewed (changes requested):
Hi there @linkztream!

Thanks for the PR, I've reviewed it, and left a comment.
Could you please take a look? 👍

linkztream committed:
Removed hidden sensors text

After changes made to the integration, sensors are no longer hidden and that text became obsolete. Removed to avoid confusion.

frenck reviewed (approved):
Thanks, @linkztream! 👍
